### PR TITLE
Added some minor clarification of BAF file gen

### PR DESCRIPTION
**Pull request by migbro**
#### Purpose

BAF file generation is described, but missing in the alignment section was gvcf output used as input to create this.
Will still need to address references used for BAF step.

#### Issue

https://github.com/AlexsLemonade/OpenPBTA-manuscript/issues/73

#### Pull review checklist

The PR will be considered ready for review when all four items have been checked.

- [x ] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [ ] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [ ] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [ ] All figures follow the [Manubot figure instructions](https://github.com/jaclyn-taroni/OpenPBTA-manuscript/blob/master/USAGE.md#figures)

### CODE REVIEW

**Comment by migbro** [comment]
I added a link for Broad references, however, I'm uncertain that the s3 link formatting is correct and may need some help on that.

**Comment by migbro** [comment]
Help! I don't know what I've done wrong.  I've tried reading the build fail logs, tried 3 different ways to edit two sentences, I'm lost :-/

File: content/03.methods.md [hunk 1]
```diff
@@ -64,7 +64,10 @@ We used BWA-MEM [@arxiv:1303.3997] v0.7.17 for alignment of paired-end DNA-seq r
 The alignment reference that we used was Homo Sapiens Human Genome (hg) version 38, patch release 12, fasta file obtained from UCSC [@url:http://hgdownload.soe.ucsc.edu/goldenPath/hg38/bigZips/].
 Alignments were further processed using following the Broad Institute's Best Practices [@url:https://software.broadinstitute.org/gatk/best-practices/workflow?id=11165] for processing BAMs in preparation for variant discovery.
 Duplicates were marked using Samblaster[@doi:10/f6kft3] v0.1.24, BAMs merged and sorted using Sambamba [@doi:10/gfzsfw] v0.6.3.
-Lastly, resultant BAMs were processing using Broad's Genome Analysis Tool Kit (GATK) [@url:https://software.broadinstitute.org/gatk/] v4.0.3.0, BaseRecalibrator submodule.
+Resultant BAMs were processing using Broad's Genome Analysis Tool Kit (GATK) [@url:https://software.broadinstitute.org/gatk/] v4.0.3.0, BaseRecalibrator submodule.
```
Comment (jashapiro) [inline content/03.methods.md]:
```suggestion
Resultant BAMs were processing using Broad's Genome Analysis Tool Kit [GATK](@url:https://software.broadinstitute.org/gatk/) v4.0.3.0, BaseRecalibrator submodule.
```

**Comment by migbro** [comment]
Thanks @jashapiro!  I was under the impression that when using the @url
syntax, that [] were needed...should I drop the @url and do a standard
markdown link style? If not, I'll just incorporate your suggestion.  Thanks
again!

**Comment by jashapiro** [comment]
> Thanks @jashapiro! I was under the impression that when using the @url syntax, that [] were needed...should I drop the @url and do a standard markdown link style? If not, I'll just incorporate your suggestion. Thanks again!

I actually have no idea if that will work...and I kind of think it won't! I didn't mean to finalize the suggestion yet! It was

**Comment by migbro** [comment]
No worries.  I went back to a traditional markdown link style - I figured the two were incompatible. If the content looks good otherwise, please approve and merge when you can.  Thanks again!

File: content/03.methods.md [hunk 2]
```diff
@@ -64,7 +64,10 @@ We used BWA-MEM [@arxiv:1303.3997] v0.7.17 for alignment of paired-end DNA-seq r
 The alignment reference that we used was Homo Sapiens Human Genome (hg) version 38, patch release 12, fasta file obtained from UCSC [@url:http://hgdownload.soe.ucsc.edu/goldenPath/hg38/bigZips/].
 Alignments were further processed using following the Broad Institute's Best Practices [@url:https://software.broadinstitute.org/gatk/best-practices/workflow?id=11165] for processing BAMs in preparation for variant discovery.
 Duplicates were marked using Samblaster[@doi:10/f6kft3] v0.1.24, BAMs merged and sorted using Sambamba [@doi:10/gfzsfw] v0.6.3.
-Lastly, resultant BAMs were processing using Broad's Genome Analysis Tool Kit (GATK) [@url:https://software.broadinstitute.org/gatk/] v4.0.3.0, BaseRecalibrator submodule.
+Resultant BAMs were processing using Broad's Genome Analysis Tool Kit [GATK] (https://software.broadinstitute.org/gatk/) v4.0.3.0, BaseRecalibrator submodule.
+Lastly, for normal/germline input, we run the GATK HaplotypeCaller submodule on the recalibrated bam, generating a gvcf file.
```
Comment (jashapiro) [inline content/03.methods.md]:
```suggestion
Lastly, for normal/germline input, we run the GATK HaplotypeCaller[@doi:10.1101/201178] submodule on the recalibrated bam, generating a gvcf file.
```

**Pull request merged**